parse_weather_data.py

```diff
@@ -0,0 +1,48 @@
+from weather_reading import WeatherReading
+
+
+class WeatherParser:
+    def parse_weather_file(self, file_path):
+        weather_readings = []
+        try:
+            with open(file_path, 'r') as weather_file:
+                header = [item.strip() for item in next(weather_file).strip().split(',')]
```

Comment:
Can we look into a reasonable way of reading this data? This data is in a defined structure and we can find many utilities around it to ease the process.